### PR TITLE
Move system-info to Temporal.now (Fixes #236)

### PR DESCRIPTION
**Pull request by pipobscure**

### CODE REVIEW

File: polyfill/lib/now.mjs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export const now = {
+    absolute: getAbsolute,
+    dateTime: getDateTime,
+    date: getDate,
+    time: getTime,
+    timeZone: getTimeZone
+};
+
+function getAbsolute() {
```
Comment (littledan) [inline polyfill/lib/now.mjs]: Nit: These functions will have the wrong name property.

File: polyfill/test/exports.mjs [hunk 2]
```diff
@@ -18,8 +18,9 @@ import * as Temporal from 'tc39-temporal';
 
 describe('Exports', () => {
   const named = Object.keys(Temporal);
-  it(`should be 13 things`, () => {
-    equal(named.length, 13);
+  console.error(named);
```
Comment (Ms2ger) [inline polyfill/test/exports.mjs]: Probably want to drop this again.

File: polyfill/lib/now.mjs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+function absolute() {
+    const ns = ES.SystemUTCEpochNanoSeconds();
+    return new Absolute(ns);
+}
+function dateTime(zone = timeZone()) {
+    zone = ES.ToTimeZone(zone);
+    const abs = absolute();
+    let dateTime = zone.getDateTimeFor(abs);
```
Comment (Ms2ger) [inline polyfill/lib/now.mjs]: Your use of let vs const confuses me :)

File: polyfill/lib/now.mjs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    return dateTime;
+}
+function date(zone) {
+    return dateTime(zone).getDate();
```
Comment (Ms2ger) [inline polyfill/lib/now.mjs]: This doesn't seem to be robust against author code messing with prototypes; I didn't quite follow if that was a goal at this point.

File: polyfill/test/exports.mjs [hunk 5]
```diff
@@ -18,8 +18,9 @@ import * as Temporal from 'tc39-temporal';
...
+  it(`should be 9 things`, () => {
```
Comment (Ms2ger) [inline polyfill/test/exports.mjs]: Wanna add `now` to the tests below?

File: polyfill/lib/now.mjs (hunk 3 above)
Comment (pipobscure) [reply polyfill/lib/now.mjs]: Me too 😄

File: polyfill/lib/now.mjs (hunk 4 above)
Comment (pipobscure) [reply polyfill/lib/now.mjs]: Should probably discuss. Though I think it's fine for now.

**Pull request merged**